enterprise/frontend/src/metabase-enterprise/database_routing/DatabaseRoutingSection/DatabaseRoutingSection.unit.spec.tsx

```diff
@@ -26,6 +26,29 @@ describe("DatabaseRoutingSection", () => {
     setup({ engine: "postgres", features: ["database-routing"] });
 
     expect(screen.getByText("Database routing")).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        "When someone views a question using data from this database, Metabase will send the queries to the destination database set by the person's user attribute. Each destination database must have identical schemas.",
+      ),
+    ).toBeInTheDocument();
+    expect(screen.getByText("Enable database routing")).toBeInTheDocument();
+    expect(screen.getByLabelText("Enable database routing")).toBeEnabled();
+  });
+
+  it("should render DatabaseRoutingSection with custom db_routing_info", () => {
+    setup({
+      engine: "bigquery-cloud-sdk",
+      features: ["database-routing"],
+      db_routing_info:
+        "Route queries on this database to a different BigQuery project or Google Cloud account based on the person's user attribute. Each BigQuery destination must have identical schemas.",
+    });
+
+    expect(screen.getByText("Database routing")).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        "Route queries on this database to a different BigQuery project or Google Cloud account based on the person's user attribute. Each BigQuery destination must have identical schemas.",
+      ),
```

Comment:
I think this is a little misleading using this text, I'd adjust the assertion to make it clearer that we're just passing through the value like 

`db_routing_info: "❤️✨CUSTOM DB ROUTING INFO ✨❤️ "`